python/tests/test_edges.py

```diff
@@ -371,7 +371,7 @@ def gen_parameters(*args,
 @pytest.mark.parametrize("source_type,sink_type",
                          [
                              pytest.param(m.Base, m.Base, id="source_base-sink_base"),
-                             pytest.param(m.Base, m.DerivedA, id="source_base-sink_derived", marks=pytest.mark.xfail),
+                             pytest.param(m.Base, m.DerivedA, id="source_base-sink_derived", marks=pytest.mark.skip),
```

Comment:
why switch to skip? does the test segfault?

Reply:
Yes, it was already marked "xfail", but newer versions of pybind have better error checking and this test was triggering an assert

Reply:
These are marked `xfail` because they should generate a runtime error telling you that you cant make an edge between these types. The reason it was failing before was due to MRC throwing the desired exception, not due to an error in pybind. What is the new pybind11 assert that is getting triggered?